### DISCUSSION TITLE
iOS and IPadOS next testflight beta availability

### DISCUSSION THREAD

**Discussion by SpyrosTheBoss** [ppy/osu#24486] [Q&A] [answered]
When is the next testflight beta gonna be opened?

Or was it open recently and almost immediately was filled up?

**Comment by SpyrosTheBoss** [comment]
Well the tablet isn't mine, and another question. Do the resets happen
Every update of lazer?

**Comment by peppy** [comment, marked as answer]
We hope to get an app store build up in the near future.

**Reply by SpyrosTheBoss** [reply]
Thank you for your answer and time Mr. Dean
